lib/std/http/method.zig

```diff
@@ -0,0 +1,17 @@
+//! HTTP Methods
+//! https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
+
+// Style guide is violated here so that @tagName can be used effectively
+/// https://datatracker.ietf.org/doc/html/rfc7231#section-4 Initial definiton
+/// https://datatracker.ietf.org/doc/html/rfc5789#section-2 PATCH
+pub const Method = enum {
+    GET,
+    HEAD,
+    POST,
```

Comment:
For later optimisations I'd suggest matching the 2 entries in the HPACK static table (https://httpwg.org/specs/rfc7541.html#static.table.definition).
`GET = 1`
`POST = 2`

Reply:
you mean 2 and 3 ?

Reply:
not sure if assigning numbers to those two would help, since `@enumToInt` would not apply to any of the others

Reply:
> you mean 2 and 3 ?

Ah yep sorry; brain fail. I edited my comment